packages/loader/container-loader/src/connectionStateHandler.ts

```diff
@@ -13,7 +13,7 @@ import { ConnectionState } from "./container";
 export interface IConnectionStateHandler {
     quorumClients: () => IQuorumClients | undefined;
     logConnectionStateChangeTelemetry:
-        (value: ConnectionState, oldState: ConnectionState, reason?: string | undefined) => void;
+    (value: ConnectionState, oldState: ConnectionState, reason?: string | undefined) => void;
```

Comment:
Nit: typically we've added a level of indent for this sort of scenario, or alternatively split params to multiline

Reply:
Fixed.